lib/services/ios-debug-service.ts

```diff
@@ -244,6 +255,14 @@ class IOSDebugService implements IDebugService {
             return inspectorPath;
         }).future<string>()();
     }
+
+    
+    private getReadyForAttachTimeout(timeoutInMilliseconds?: number): number {
+        let timeout = timeoutInMilliseconds || this.$utils.getMilliSecondsTimeout(IOSDebugService.TIMEOUT_SECONDS);
+        let readyForAttachTimeout = timeout / 10 ;
+        let defaultReadyForAttachTimeout = 5000;        
+        return readyForAttachTimeout > defaultReadyForAttachTimeout ? readyForAttachTimeout : defaultReadyForAttachTimeout;
```

Comment:
You can use (Math.max)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max] here